jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -69,33 +69,14 @@ node {
 
         stage("gen-assembly") {
             withEnv(['KUBECONFIG=/home/jenkins/kubeconfigs/art-publish.app.ci.kubeconfig']) {
-                nightly_args = ""
-                for (nightly in commonlib.parseList(params.NIGHTLIES)) {
-                    nightly_args += " --nightly ${nightly.trim()}"
-                }
-                cmd = "--group openshift-${BUILD_VERSION} release:gen-assembly --name ${ASSEMBLY_NAME} from-releases ${nightly_args}"
-                if (params.CUSTOM) {
-                    cmd += ' --custom'
-                    if ((params.IN_FLIGHT_PREV && params.IN_FLIGHT_PREV != 'none') || params.PREVIOUS) {
-                        error("Specifying IN_FLIGHT_PREV or PREVIOUS for a custom release is not allowed.")
-                    }
-                }
-                else {
-                    if (params.IN_FLIGHT_PREV && params.IN_FLIGHT_PREV != 'none') {
-                        cmd += " --in-flight ${IN_FLIGHT_PREV}"
-                    }
-                    if (params.PREVIOUS) {
-                        for (previous in params.PREVIOUS.split(',')) {
-                            cmd += " --previous ${previous.trim()}"
+            withCredentials([string(credentialsId: 'ashwin-github-token', variable: 'GITHUB_TOKEN')]) {
+                dir("pyartcd/pyartcd/pipelines"){
+                            sh "python gen_assembly.py --nightlies='${params.NIGHTLIES}' --in_flight_prev=${params.IN_FLIGHT_PREV} --build_version=${BUILD_VERSION} --assembly_name=${params.ASSEMBLY_NAME}"
```

Comment:
For param names let's do hyphens instead of underscore, so 
`--in-flight-prev`,  and keep them concise `--version`, `--assembly`

Reply:
Okay!

Reply:
Done!